src/main/fox_hud.c

```diff
@@ -4668,10 +4937,446 @@ void func_800933D8(f32 x, f32 y, f32 z, f32 arg3) {
+
+            player->camEye.x = D_80177978 = -3061.2f;
+            player->camEye.y = D_80177980 = 1745.9f;
+            player->camEye.z = D_80177988 = -5786.0f;
+
+            player->camAt.x = D_801779A0 = 300.0f;
+            player->camAt.y = D_801779B8 = 2200.0f;
+            player->camAt.z = D_801779C0 = -5700.0f;
+
+            D_801616A0.x = 5.18f;
+            D_801616A0.y = 124.17f;
+            D_801616A0.z = 0.00f;
+
+            actor->unk_0F4.y = 30.0f;
+
+            D_80177A48[5] = 14.0f;
+            D_80177A48[0] = 0.2f;
+            D_80177A10[6] = 0;
+            D_80177A10[7] = 0;
+            D_801616B0.x = D_801616B0.y = D_801616B0.z = 0.0f;
+            D_80177A48[7] = 0.0f;
+
+        case 1:
+            if ((player->unk_208 == 0) && (D_80177A80 < 250)) {
+                i = Rand_ZeroOne() * 64.0f;
+                if (i < 32) {
+                    for (j = 0; j < 4; j++) {
+                        src.x = D_800D2320[i].x;
+                        src.y = D_800D2320[i].y + (Rand_ZeroOne() * 12.0f);
+                        src.z = D_800D2320[i].z;
+
+                        Matrix_Translate(gCalcMatrix, actor->obj.pos.x, actor->obj.pos.y, actor->obj.pos.z, 0);
+                        Matrix_RotateY(gCalcMatrix, M_DTOR * 30.0f, 1);
+                        Matrix_MultVec3f(gCalcMatrix, &src, &dest);
+
+                        x = dest.x;
+                        y = dest.y;
+                        z = dest.z;
+
+                        func_800933D8(x, y, z, 20.0f);
+
+                        i = (i + 1) % 32;
+                    }
+                } else {
+                    i -= 32;
+                    for (j = 0; j < 4; j++) {
+                        src.x = -D_800D2320[i].x;
+                        src.y = D_800D2320[i].y + (Rand_ZeroOne() * 12.0f);
+                        src.z = D_800D2320[i].z;
+
+                        Matrix_Translate(gCalcMatrix, actor->obj.pos.x, actor->obj.pos.y, actor->obj.pos.z, 0);
+                        Matrix_RotateY(gCalcMatrix, M_DTOR * 30.0f, 1);
+                        Matrix_MultVec3f(gCalcMatrix, &src, &dest);
+
+                        x = dest.x;
+                        y = dest.y;
+                        z = dest.z;
+
+                        func_800933D8(x, y, z, 20.0f);
+
+                        i = (i + 1) % 32;
+                    }
+                }
+                player->unk_208 = (s32) (Rand_ZeroOne() * 3.0f) + 1;
+            } else {
+                player->unk_208--;
+            }
+
+            if (D_80177A10[6] < 3) {
+                i = D_80177A10[6];
+                stepSize = D_80177A48[5];
+
+                src.x = D_80177978;
+                src.y = D_80177980;
+                src.z = D_80177988;
+
+                dest.x = D_800D22D8[i].x;
+                dest.y = D_800D22D8[i].y;
+                dest.z = D_800D22D8[i].z;
+
+                if (Math_PursueVec3f(&src, &dest, &D_801616A0, stepSize, 0.4f, 4.8f, stepSize)) {
```

Comment:
I was wondering where this was used. Any idea what the vectors might represent?

Reply:
I haven't looked at it yet.